### PR TITLE
feat(content): thermoblade

### PR DESCRIPTION
**Pull request by yay855**
Added in the thermoblade, a utility knife that heats itself using an atomic battery. It can show up almost anywhere a hunting knife can (at a lower chance), except on police since it's legally questionable. It can be crafted yourself using a knife, electronics supplies, and a plutonium cell if you have 6 electronics and a lab recipe book or 8 electronics otherwise.

## Purpose of change

There are atomic powered cooking items ingame, but none of them are very portable, and the atompot itself isn't craftable, only the atomic coffeemaker.

## Describe the solution

The thermoblade is directly inspired by the item of the same name from Subnautica, and functionally it's a hunting knife with higher damage and cooking utility; it's an atompot without the pot, a one-tool cauterizing utility, and a decent backup weapon if you don't use it enough to damage it.

## Testing

Since this is an endgame tool that requires lab books or 8 electronics to craft, it deals more damage per hit than a combat knife and has a unique technique that deals additional heat damage. It is otherwise solely useful for players who want an infinite source of heat other than the atompot; it seems fairly balanced to me, but I welcome suggestions.

### CODE REVIEW

File: data/json/techniques.json [hunk 1]
```diff
@@ -157,6 +157,14 @@
     "messages": [ "You disarm %s using your whip", "<npcname> disarms %s using their whip" ],
     "description": "Unwield target's weapon"
   },
+  {
+    "type": "technique",
+    "id": "tec_heatblade",
+    "name": "thermal stab",
+    "messages": [ "You stab %s, melting their flesh", "<npcname> stabs %s, melting their flesh" ],
+    "melee_allowed": true,
+    "flat_bonuses": [ { "stat": "damage", "type": "heat", "scale": 4.0 } ]
```
Comment (chaosvolt) [inline data/json/techniques.json:166]: Hmm. Free heat damage for no power use. I wonder, would it make more sense to have this tec be assigned to an active state, have the active state use an on-relic effect to deal bonus fire damage, or just use the mundane old fireweapon code for the old flaming weapons that've been obsolete for forever?

Comment (yay855) [reply data/json/techniques.json]: Yep, atomic, and the technique has now been removed in favor of the FLAMING flag.

**Review by chaosvolt — APPROVED** [review]
images

**Comment by VissValdyr** [comment]
We should consider making a atomic pot recipe as well. It should be uncraftable, so making a crafting recipe based on it (with the recipe being in the middle of the blade and the coffeemaker is to be considered.

**Pull request merged**